x/oracle/keeper/keeper.go

```diff
@@ -181,6 +183,9 @@ func (k Keeper) GetVotePenaltyCounter(ctx sdk.Context, operator sdk.ValAddress)
 
 // SetVotePenaltyCounter updates the # of vote periods missed in this oracle slash window
 func (k Keeper) SetVotePenaltyCounter(ctx sdk.Context, operator sdk.ValAddress, missCount uint64, abstainCount uint64) {
+	defer metrics.SetOracleVotePenaltyCount(missCount, "miss")
```

Comment:
shouldn't we also be labeling on the validator address so the vote penalties are partitioned by val Addr

Reply:
ah yeah forgot,t hanks!